charts/jx-pipelines-visualizer/templates/deployment.yaml

```diff
@@ -94,4 +101,9 @@ spec:
       {{- end }}
       {{- with .Values.pod.schedulerName }}
       schedulerName: {{ tpl . $ | trim }}
-      {{- end }}
\ No newline at end of file
+      {{- end }}
+      volumes:
+      - name: secrets-git
+        secret:
+          defaultMode: 420
+          secretName: tekton-git
```

Comment:
@rawlingsj can we make it optional?
for example on our v2 cluster we don't have such a secret. Also people may install the pipelines visualizer in a different namespace, without this secret.
thanks!